README.rst

```diff
@@ -512,6 +512,18 @@ setting ``DEFAULT_PRIORITY``. Integration with asynchronous email backends
         'DEFAULT_PRIORITY': 'now'
     }
 
+Override recipient
+----------------
+
+The default is disabled. If you went redirect all sended mails, for testing mail rendering in developpement period
+
+.. code-block:: python
+
+    # Put this in settings.py
+    POST_OFFICE = {
+        'OVERRIDE_RECIPIENTS': 'to@exemple.com'
```

Comment:
In addition to a single email address, I think we should also allow this configuration option to support a list of addresses, such as `['a@example.com', 'b@example.com']`

Reply:
Commit #a5dd6d8